### DISCUSSION TITLE
Material help PLEASE

### DISCUSSION THREAD

**Discussion by Wackanim** [BadDogSkyrim/PyNifly#223] [Q&A]
I've been trying to import the Pip-boy model from Fallout four but every time I do, I shows up as pure black in the rendered view. I'm sure you've told people how to do this before, but I couldn't find anything like this in the discussions. Mainly, I just want a basic rundown on how to correctly import the materials. I'm using BAE to extract the files from fallout, but I've just gotten stuck here. I'd be very grateful for some assistance.

![untitled](https://github.com/BadDogSkyrim/PyNifly/assets/149446906/4498dece-ef24-4eed-894b-cdbc721d1d5a)
If this has been explained, please point me to where it was solved.

**Comment by BadDogSkyrim** [comment]
Have you got the latest version? I've been mucking with the shaders and the pipboy is coming in properly for me.

**Reply by Daedra6102** [reply]
https://forums.nexusmods.com/topic/13509406-some-models-appear-black-when-importing-into-blender/

Dont have a solution but it has something to do with file pathing. I am having same exact issue with skyrim models but some reason modded models work fine.

**Comment by BadDogSkyrim** [comment]
I responded in the referenced thread (on nexusmods). Let me know if that doesn't do it for you.

**Reply by Daedra6102** [reply]
Got it working, some reason the texture folder got corrupted so I had to extract it again for it which fixed it.
